src/libs/Navigation/CustomActions.js

```diff
@@ -1,6 +1,42 @@
 import {CommonActions, StackActions, DrawerActions} from '@react-navigation/native';
 import lodashGet from 'lodash/get';
 
+/**
+ * Go back to the Main Drawer
+ * @param {Object} navigationRef
+ */
+function navigateBackToDrawer(navigationRef) {
```

Comment:
I'm having trouble with this name because it implies we are calling this when the drawer is not active. It looks like something I would call if I was not on the drawer to take me back to the drawer. However, if you look we also have logic for handling when the drawer _is_ active.

Ultimately, I find handling both responsibilities confusing and would love to figure out an alternative.